lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -40,6 +40,36 @@ async function getFileContent(
   return fileContent;
 }
 
+function hasAny(set: Set<string>, targets: Iterable<string>): boolean {
+  for (const target of targets) {
+    if (set.has(target)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+function getManagersForPackageFiles(
+  packageFiles: FileAddition[],
+  managerPackageFiles: Record<string, Set<string>>,
+): Set<string> {
+  const packageFileNames = packageFiles.map((packageFile) => packageFile.path);
+  return new Set(
+    Object.keys(managerPackageFiles).filter((manager) =>
+      hasAny(managerPackageFiles[manager], packageFileNames),
+    ),
+  );
+}
+
+function getPackageFilesForManager(
+  packageFiles: FileAddition[],
```

Comment:
```suggestion
  packageFiles: FilePath[],
```

Reply:
This returns a filtered version of the list of `FileAddition`s, so it needs to take the whole object.

Reply:
make function generic, should work as expected 🤗